pkg/status/status_test.go

```diff
@@ -0,0 +1,64 @@
+package status
```

Comment:
let's add an unit test for the case where the health state is nil and make sure the reverse conversion is returning NOT_KNOWN.